appveyor.yml

```diff
@@ -4,7 +4,7 @@ branches:
   - master
 image: Visual Studio 2022

```

Comment:
Je zou 'Visual Studio 2022 Preview' kunnen proberen.

Reply:
Dat is idd een goeie

Reply:
https://github.com/progressonderwijs/ProgressOnderwijsUtils/pull/955